Monika After Story/game/script-greetings.rpy

```diff
@@ -371,3 +371,52 @@ label monikaroom_greeting_post:
         set_keymaps()
     return
 
+label_greeting_youarereal:
```

Comment:
the correct syntax for labels is:
```
label this_is_label
```

Please fix your label syntax as well as the indentation for this label